src/Parser/VengeanceDemonHunter/Modules/Statistics/Spells/DemonSpikes.js

```diff
@@ -21,7 +26,7 @@ class DemonSpikes extends Module {
         icon={<Icon icon="ability_demonhunter_demonspikes" alt="Demon Spikes" />}
```

Comment:
Please use `SpellIcon` here to users can consistently hover the icon for tooltips about the spells to learn more about them.